telemetry/specs/telemetry_report.mdk

```diff
@@ -473,48 +548,38 @@ particular deployment.
         [github.com/p4lang/switch](https://github.com/p4lang/switch/blob/master/p4src/includes/drop_reason_codes.h).
 
     See the INT specification [^INT] for definitions of the remaining metadata.
+    
+* Domain Specific ID (16b)
+  : The unique ID of the INT Domain
+
+* DSMdbit (16b): Domain Specific Md Bits
+  : Bitmap that indicates which optional domain specific metadata is 
+    present in the record of the telemetry report. Each bit represents 
+    4 octets of domain specific optional metadata.
```

Comment:
I don't know where to specify this since it falls in between INT Data Plane and Telemetry Report. I guess this is as good a place as any.

When using INT-MD or INT-MX, if the Domain Specific ID does not match any Domain ID known to this node, then the node may either:

- Copy the INT Hop ML to the Telemetry Report MD Length, copy the INT DS Instruction to the Telemetry Report DSMdbit field, and pad the Variable Optional Baseline & DS Metadata with the special all-ones reserved value out to MD Length, or
- Set the Telemetry Report DSMdbit field to zero and rederive the Telemetry Report MD Length from RepMdBits.

Reply:
Added